src/system-health/health_checker/utils.py

```diff
@@ -1,25 +1,25 @@
-import subprocess
-
-
-def run_command(command):
-    """
-    Utility function to run an shell command and return the output.
-    :param command: Shell command string.
-    :return: Output of the shell command.
-    """
-    try:
-        process = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE)
-        return process.communicate()[0].encode('utf-8')
-    except Exception:
-        return None
-
-
-def get_uptime():
-    """
-    Utility to get the system up time.
-    :return: System up time in seconds.
-    """
-    with open('/proc/uptime', 'r') as f:
-        uptime_seconds = float(f.readline().split()[0])
-
-    return uptime_seconds
+import subprocess
+
+
+def run_command(command):
+    """
+    Utility function to run an shell command and return the output.
+    :param command: Shell command string.
+    :return: Output of the shell command.
+    """
+    try:
+        process = subprocess.Popen(command, shell=True, universal_newlines=True, stdout=subprocess.PIPE)
+        return process.communicate()[0].encode('utf-8')
```

Comment:
should remove "encode", previously I found that it needs to add "decode" when using the return value of this function is because of this.

Reply:
Done. Thanks!